index.js

```diff
@@ -124,5 +149,16 @@ module.exports = {
     });
 
     await this.updateDeps(options);
+
+    const lintFix = require('ember-cli/lib/utilities/lint-fix');
```

Comment:
lint:fix wasn't running at all before and now it is (and is kinda required, since version drift could change how the output of files end up)